caffe2/python/onnx/tests/onnx_backend_test.py

```diff
@@ -157,6 +157,12 @@
                      '|test_conv_with_autopad_same_.*'
                      ')')
 
+# Temporarily skip some ONNX backend tests due to updates in opset 15.
+backend_test.exclude('(test_bernoulli_*'  # new operator added
+                     '|test_castlike_*'  # new operator added
+                     '|test_optional_*'  # new operator added
+                     ')')
```

Comment:
Some[ `Optional`](https://github.com/onnx/onnx/blob/master/docs/Operators.md#Optional) operators updated in opset 15 also should be added?

Reply:
added, although this will not be needed as the onnx submodule part will be covered by #63716.